packages/express/src/middleware/index.ts

```diff
@@ -45,7 +51,7 @@ export function expressMiddleware(appsignal: Client): RequestHandler {
           span.setName(`${method} ${baseUrl}${req.route.path}`)
         }
 
-        span.setSampleData("params", { ...params, ...query })
+        span.setSampleData("params", { ...params, ...query, ...body })
```

Comment:
I think we should check if the `body` is an object or not. And skip if it it's not. According to the types it can be `any`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ec3525c11ef9f60e731b9a0e9bd64bfc4bf9fdab/types/express/index.d.ts#L115